lib/slim-rails/register_engine.rb

```diff
@@ -30,6 +30,7 @@ def self._register_engine(config)
         config.assets.configure do |env|
           if env.respond_to?(:register_transformer) && Sprockets::VERSION.to_i > 3
             env.register_mime_type 'text/slim', extensions: ['.slim', '.slim.html']#, charset: :html
+            env.register_mime_type 'application/javascript', extensions: ['.js.jst.ejs.slim']
```

Comment:
This seems to be very specific and tailored for your use case, but I think that if we're going add this, then let's do it in a more general form, how about:

```ruby
env.register_mime_type 'application/javascript', extensions: %w[.js.slim .jst.slim .ejs.slim]
```

Reply:
Happy to do so. However I did realize that it is not working as intended. Will continue investigation and report back once solved.